monkey/monkey_island/cc/ui/src/components/report-components/security/StolenCredentialsTable.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useEffect, useState} from 'react';
+import ReactTable from 'react-table'
+import IslandHttpClient, {APIEndpoint} from '../../IslandHttpClient';
+import LoadingIcon from '../../ui-components/LoadingIcon';
+import {reformatSecret} from '../credentialParsing';
+import _ from 'lodash';
+
+
+const columns = [
+  {
+    Header: 'Stolen Credentials',
+    columns: [
+      {Header: 'Username', accessor: 'username'},
+      {Header: 'Type', accessor: 'title'}
+    ]
+  }
+];
+
+const pageSize = 10;
+
+const StolenCredentialsTable = () => {
+
+  const [credentialsTableData, setCredentialsTableData] = useState(null);
+
+  useEffect(() => {
+    IslandHttpClient.get(APIEndpoint.stolenCredentials)
```

Comment:
`SecurityReport` is already pulling this data. Would it be better to pass that info to this component, rather than fetch the data twice?